cobc/ppparse.y

```diff
@@ -1368,6 +1373,30 @@ leap_second_directive:
 | OFF
 ;
 
+imp_directive:
+  _include imp_include_sources
+  {
+	cb_include_file_list = ppp_list_append(cb_include_file_list, $2);
```

Comment:
You won't set the option here but in scanner.l

the rule of thumb is: both of those should resolve in the same output

* `cobc test.cob -o test.c` 
* `cobc test.cob -o test.i && cobc test.i -o test.c`

but as-is the directive is lost.

Also; it is likely reasonable to have `cb_include_file_list_directive` as you can then free it and set it to NULL between the sources:

* `cobc --include "<windows.h>" prog1.cob prog2.cob --save-temps` should have the include in both prog1.c + prog2.c
* `cobc prog1.cob prog2.cob --save-temps`  with `>> IMP INCLUDE "<windows.h>"` in the first program should not have it included in prog2,c

Reply:
> Also; it is likely reasonable to have cb_include_file_list_directive as you can then free it and set it to NULL between the sources

I believe I can use `cb_include_file_list`? Since it has been introduced for that purpose in Fabrice's previous PR. Do we really need two lists?